internal/keycloakb/accreditationsmodule.go

```diff
@@ -38,7 +33,7 @@ type AccredsKeycloakClient interface {
 
 // AdminConfigurationDBModule interface
 type AdminConfigurationDBModule interface {
-	GetAdminConfiguration(context.Context, string) (configuration.RealmAdminConfiguration, error)
+	//GetAdminConfiguration(context.Context, string) (configuration.RealmAdminConfiguration, error)
```

Comment:
Does it mean that this interface is not used anymore? (or else, it is an empty interface)

Reply:
Good catch. I missed it during code cleaning